spec/spec_helper.rb

```diff
@@ -20,8 +20,8 @@
     @ca_key_pass = 'aaaa'
     @ca_cert = File.join(File.dirname(__FILE__), 'cacert.pem')
 
-    @client_id = "sexyhorse"
-    @client_email = "sexyhorse@zyp.io"
+    @client_id = 'sexyhorse'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.